src/sentry/scim/endpoints/users.py

```diff
@@ -0,0 +1,211 @@
+from sentry.api.endpoints.organization_member_index import OrganizationMemberSerializer
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.organization_member import OrganizationMemberSCIMSerializer
+from sentry.api.validators import AllowedEmailField
+from sentry.models import AuditLogEntryEvent, AuthIdentity, OrganizationMember
+from sentry.signals import member_invited
+
+from .utils import SCIM_API_ERROR, SCIMEndpoint, parse_filter_conditions
+
+ERR_ONLY_OWNER = "You cannot remove the only remaining owner of the organization."
+
+SCIM_404_USER_RES = {
+    "schemas": [SCIM_API_ERROR],
+    "detail": "User not found.",
+}
+
+SCIM_409_USER_EXISTS = {
+    "schemas": ["urn:ietf:params:scim:api:messages:2.0:Error"],
+    "detail": "User already exists in the database.",
+}
+
+
+class SCIMUserSerializer(serializers.Serializer):
+    email = AllowedEmailField(max_length=75, required=True)
+    role = serializers.ChoiceField(choices=roles.get_choices(), required=True)
+    sendInvite = serializers.BooleanField(required=False, default=True, write_only=True)
+
+    def validate_email(self, email):
+        queryset = OrganizationMember.objects.filter(
+            Q(email=email) | Q(user__email__iexact=email, user__is_active=True),
+            organization=self.context["organization"],
+        )
+
+        if queryset.filter.exists():
+            raise serializers.ValidationError("User already exists in the database.")
+        return email
+
+
+class OrganizationSCIMUserDetails(SCIMEndpoint, OrganizationMemberDetailsEndpoint):
+    def _get_member(self, organization, member_id):
+        try:
+            member = OrganizationMember.objects.get(organization=organization, id=member_id)
+        except OrganizationMember.DoesNotExist:
+            raise ResourceDoesNotExist
+        except AssertionError as error:
+            if str(error) == "value too large":
+                raise ResourceDoesNotExist
+            raise error
+        return member
+
+    def get(self, request, organization, member_id):
+        try:
+            member = self._get_member(organization, member_id)
+        except ResourceDoesNotExist:
+            return Response(SCIM_404_USER_RES, status=404)
+
+        context = serialize(member, serializer=OrganizationMemberSCIMSerializer())
+        return Response(context)
+
+    def patch(self, request, organization, member_id):
+        try:
+            member = self._get_member(organization, member_id)
+        except ResourceDoesNotExist:
+            return Response(SCIM_404_USER_RES, status=404)
+
+        for operation in request.data.get("Operations", []):
+            # we only support setting active to False which deletes the orgmember
+            if operation["value"]["active"] is False:
```

Comment:
Could you also get `0` or is the spec pretty strict?

Reply:
spec is strict, active is defined as a json boolean